janitor/xarray/utils.py

```diff
@@ -0,0 +1,43 @@
+"""
+Helpers to facilitate creating new XArray methods
```

Comment:
Can I just quickly check, are the contents of this submodule present in `pandas-flavor`? If so, then do we need them here?

Reply:
Yeah. Blocked by https://github.com/Zsailer/pandas_flavor/issues/13#issuecomment-552208952 for now. Swapping that out when a release is cut will be seamless, I think.

Reply:
No problem, ok, let's leave it in for now. I'll set a limit of like a few days for now, hopefully we have a new release from @Zsailer on `pandas-flavor`. If not, we can swap out in another PR.

Reply:
pandas_flavor v0.2.0 is now available on PyPI. Working on the [conda-forge package](https://github.com/conda-forge/pandas_flavor-feedstock/pull/2/checks?check_run_id=296998802)...

Reply:
Removed my method registration code and moved to using pf’s